label_studio/core/api_permissions.py

```diff
@@ -6,8 +6,9 @@ def has_object_permission(self, request, view, obj):
         return obj.has_permission(request.user)
 
 
-class HasOwnerPermission(BasePermission):
+class MemberHasOwnerPermission(BasePermission):
     def has_object_permission(self, request, view, obj):
-        if not request.user.own_organization or obj.active_organization != request.user.active_organization:
+        if request.method == 'DELETE' and not request.user.own_organization:
```

Comment:
:wave: just took a glance at this - I think we shouldn't use http methods in permissions implementations. I suspect one could still perform deletes by making a PATCH request with `deleted_at` to the `OrganizationMembershipAPI` for example.